src/components/SideBar/SideBarItem.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, Icon, Text, Flex } from "@chakra-ui/react";
+
+const SideBarItem = (props: any) => {
+  return (
+    <a data-testid={props.item_text}>
+      <Box w="100%" p="32px 7px">
```

Comment:
Box has an props called "as", you can use as to set this sidebar item as an button
<img width="857" alt="image" src="https://user-images.githubusercontent.com/61776449/168308110-cdf7388b-a1f0-49f0-a77c-53e2b428323c.png">
https://chakra-ui.com/docs/components/layout/box

Reply:
![image](https://user-images.githubusercontent.com/61776449/168308456-549f8560-5d27-4d23-a251-d078aa1de3fc.png)
Since we are now replicating the editor of Canva, I think you can take a look at this style